Assets/Scripts/HealthBar.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HealthBar : MonoBehaviour
+{
+    public GameObject Entity;
```

Comment:
Hoe werkt dit precies? Bij mij is deze niet gezet in de editor en dat zie ik ook niet in de code gebeuren

Reply:
nvm

Reply:
In de gameobject zelf